### ISSUE TITLE
Media Manager - New Folder Focus

### ISSUE THREAD

**Issue by bradymccain** [joomla/joomla-cms#37452]
### Steps to reproduce the issue
Open Media Manager
Click "Create New Folder"

### Expected result
The new folder field should have focus when opened so that the user doesn't have to click on the field before typing.

### Actual result
The new folder field does not have focus.

### System information (as much as possible)
Joomla 4.1.2
PHP 7.4.28
Google Chrome 100.0.4896.60

**Comment by drmenzelit** [comment]
Closing as it is a duplicate from #36881 and there is a PR #36892
